### DISCUSSION TITLE
Streaming videos from Vimeo wont load if they have audio

### DISCUSSION THREAD

**Discussion by brunomartelli** [mozilla/hubs#2859] [General Questions & Discussion] [answered]
causes an error, video won't load or play **_failed to init decoder_** crashes Spoke, crashes Firefox, windows 10, PC

**Comment by brunomartelli** [comment]
Tried with several different audio formats AAC, MPEG various sample rates bitrates etc.

**Comment by fluidodigital** [comment]
Hi all same problem here, so I tried to put a link to the video and it works also with sound. You cannot define the dimension but the video works and also loops. I linked a private video.

**Comment by brunomartelli** [comment]
Hi i now can get videos to play in a HUB room, not on spoke which makes dev difficult..

**Reply by davegoopot** [reply]
Can you post an example video link that doesn't work?

**Reply by BonzoMartini** [reply]
Try any vimeo video with sound.

**Reply by davegoopot** [reply]
This example works for me: https://vimeo.com/453295163 

Displays in Spoke, but doesn't play. I see the first frame still in Spoke.

Displays and plays, with sound, in a Hub using the Spoke scene.

**Comment by emclaren** [comment]
Linking this related discussion -  https://github.com/mozilla/hubs/discussions/2849

**Comment by emclaren** [comment, marked as answer]
This appears to be a bug being tracked here: https://github.com/mozilla/Spoke/issues/964
